lib/src/dart_dev_cli.dart

```diff
@@ -101,7 +101,7 @@ Future _run(List<String> args) async {
 
   ArgResults env;
   try {
-    env = _parser.parse(args);
+    env = tryArgsCompletion(args, _parser);
```

Comment:
So does this just try add support for parsing incomplete args? Like if someone were to accidentally run `ddev form` - it would run `ddev format`?